pkg/operator/starter.go

```diff
@@ -194,6 +197,11 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		webhookHooks,
 	)
 
+	priorityClass := defaultPriorityClass
+	if isHyperShift {
+		priorityClass = hypershiftPriorityClass
+	}
+
```

Comment:
Maybe get rid of this if statement and reuse the one below?

Reply:
Fixed